cmd/juju/model/switchgeneration.go

```diff
@@ -0,0 +1,116 @@
+// Copyright 2018 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package model
+
+import (
+	"fmt"
+
+	"github.com/juju/cmd"
+	"github.com/juju/errors"
+	"github.com/juju/gnuflag"
+
+	"github.com/juju/juju/api/modelgeneration"
+	jujucmd "github.com/juju/juju/cmd"
+	"github.com/juju/juju/cmd/modelcmd"
+)
+
+const (
+	switchGenerationSummary = "Switch to the given generation."
+	switchGenerationDoc     = `
+Switch to the given generation, either current or next. 
```

Comment:
there is a card to refine the help text here: https://trello.com/c/Kff2V2LS/924-refine-text-for-switch-generation-cmd-help